### ISSUE TITLE
Custom script only runs ONE TIME (when i save it) not ALWAYS . Help please 

### ISSUE THREAD

**Issue by JonCSgr** [RocketChat/Rocket.Chat#20792]
### Steps to reproduce:

1.  Homepage
2.  Nothing script should autoexecute

### Expected behavior:

Autlog you in 
### Actual behavior:
Nothing happens 

### Server Setup Information:

- Version of Rocket.Chat Server:  Latest
- Operating System: Latest
- Deployment Method: Latest
- Number of Running Instances: 1 
- DB Replicaset Oplog: 
- NodeJS Version: Latest
- MongoDB Version: Latest

### Client Setup Information

- Desktop App or Browser Version: Browser Firefox latest
- Operating System: Ubuntu 16.04

### Additional context

So basically i have set up Custom Script for Logged Out Users like this 
`document.getElementsByClassName(“rc-button rc-button–secondary external-login keyclock”)[0].click();`

When i press save on rocket.chat it works and the script gets executed but ONLY one time when i press save . Here is the console .log 
![image](https://user-images.githubusercontent.com/61245954/108195149-14fdc680-7120-11eb-8d91-053978eefbdd.png)

**Comment by verbeckii** [comment]
I'm sorry for disturbing this topic
but how do you solve this?
Actually, I faced the same issue - I need to click "Keycloak Login Button" by custom script when 

and this is also doesn't work fo me
> Custom Script for Logged Out Users like this
> document.getElementsByClassName(“rc-button rc-button–secondary external-login keyclock”)[0].click();
